docs/src/main/asciidoc/elasticsearch.adoc

```diff
@@ -413,3 +413,5 @@ Accessing an Elasticsearch cluster from a low level or a high level client is ea
 == Configuration Reference
 
 include::{generated-dir}/config/quarkus-elasticsearch-restclient-lowlevel.adoc[opts=optional, leveloffset=+1]
+
+You can also programmatically apply additional configuration to the client by implementing a `RestClientBuilder.HttpClientConfigCallback` and annotating it with `ElasticsearchClientConfig`. You may provide multiple implementations and configuration provided by each implementation will be applied in a randomly ordered cascading manner.
```

Comment:
This section really is about listing configuration properties, it's a bit odd to add text at the bottom, and people are likely to miss it.

I would move this text to the "Configuring Elasticsearch" section higher in this document?

Reply:
Sure thing, should I create a new section like "Programmatically Configuring Elasticsearch" right below the "Configuring Elasticsearch" section as in the [hibernate guide](https://quarkus.io/guides/hibernate-orm), or should I just continue within the Configuring Elasticsearch section? Also I was a bit unsure whether I should include some code examples or not or maybe simply include a link reference to a [here](https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/_encrypted_communication.html) for some sample usage from elasticsearch documentation. Could you please advise on that as well?

Reply:
Not sure what the rules are exactly, in some cases you have to update the quickstart as well... @loicmathieu, @gsmet , any opinion?

Reply:
Quickstart is 60/40. I don't think this customization should be there.

As for adding it to the existing or creating a new one, your choice ;)

Reply:
Any suggestions on including code samples or links as I mentioned earlier? Would it be ok if I include the "Encrypted communication" example from the Elasticsearch documentation here?

Reply:
Yes, sounds good as it's the example on the Elasticsearch doc.

Reply:
Cheers, I will update the document as per your suggestions.